OnBoardingCalculator/app/src/main/java/com/globant/onboardingcalculator/mvp/presenter/CalculatorPresenter.java

```diff
@@ -2,12 +2,25 @@
 
 import com.globant.onboardingcalculator.mvp.model.CalculatorModel;
 import com.globant.onboardingcalculator.mvp.view.CalculatorView;
-import com.globant.onboardingcalculator.utils.Constants;
+
+import java.text.DecimalFormat;
+
+import static com.globant.onboardingcalculator.utils.Constants.DECIMAL_POINT;
+import static com.globant.onboardingcalculator.utils.Constants.EMPTY_CHAR;
+import static com.globant.onboardingcalculator.utils.Constants.EMPTY_STRING;
+import static com.globant.onboardingcalculator.utils.Constants.NUMBER_ZERO;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_DIVIDE;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_MULTIPLY;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_PLUS;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_SUBSTRACTION;
+import static java.lang.Double.parseDouble;
+
 
 public class CalculatorPresenter {
 
     private CalculatorModel model;
     private CalculatorView view;
+    private DecimalFormat decimalFormat = new DecimalFormat("#.##");
```

Comment:
you should not harcode, you can use a Utils class

Reply:
Done!